src/components/FormDrawer.tsx

```diff
@@ -15,6 +15,7 @@ import Form from './Form';
 
 export interface IFormDrawerProps extends ISharedComponentProps, ISharedFormProps {
   isVisible: SmartBool;
+  onCancel: () => void;
```

Comment:
`onCancel?: () => void;` shouldn't that be optional ?

Reply:
For every component but FormDrawer, which will ALWAYS show a [X] at the top.